pkg/apis/etcd/v1beta2/backup.go

```diff
@@ -42,4 +42,7 @@ type S3Source struct {
 	//
 	// AWSSecret overwrites the default etcd operator wide AWS credential and config.
 	AWSSecret string `json:"awsSecret,omitempty"`
+
+	// AWSEndpoint
+	AWSEndpoint string `json:"awsEndpoint,omitempty"`
```

Comment:
this should be just endpoint? we also need to document this better.

empty means aws s3, which is by default. you can fill in endpoints if you wish to use minion or some other compatible provider?